tests/integration/plugins/test_python_plugin.py

```diff
@@ -106,9 +109,10 @@ class ExpectedValues:
     VALUES_FOR_HOST = RELEASE_TO_VALUES["22.04"]
 
 
+@pytest.mark.parametrize("plugin_name", ["python", "poetry"])
 @pytest.mark.parametrize("base", tuple(UBUNTU_BASES))
-def test_python_plugin_ubuntu(base, tmp_path, run_lifecycle):
-    project = create_python_project(base=base)
+def test_python_plugin_ubuntu(base, tmp_path, run_lifecycle, plugin_name):
```

Comment:
This is very creative, but not the dangerous sort of "clever code" I fear. Very nice!